### PR TITLE
[website] refactor(components): trim syntax for icons, svgs.

### PR DESCRIPTION
**Pull request by TylerAPfledderer**
_Updated version of #26739. cc @corwintines for primary approval and merging by request_

Trims the syntax for all of the svgs under `components/icons` and `components/svgs`.

## Current Rendering
The icons currently render a nested `svg` due to this element being added in the `path` prop, but `createIcon` renders the element.

## New Rendering
- Nested svgs are removed from `path` prop
- Single paths are moved to the `d` prop. `fill='currentColor'` here is rendered by default.
- For some of the icons, `width` and `height` are removed because of implementation where only the `viewBox` is needed
  - By default, the icons are rendered with `width: '1em'` and `height: '1em`'  so that `fontSize` can also be used.
- The `defaultProps` prop is used for when the width, height, color, or `fill: 'none'` need to be supplied to the svg (and removes the  added level of abstraction in exporting a layered `Icon` component!)

### CODE REVIEW

File: src/components/UI/svgs/GopherHomeNodes.tsx
Comment (corwintines) [inline src/components/UI/svgs/GopherHomeNodes.tsx:1]: @TylerAPfledderer I think there was a mistake here. Looks like the paths match up with `GopherHomeLinks`. As well as the naming changed from `GopherHomeNodes` to `GopherHomeLinks` here too.

**Review by corwintines — CHANGES_REQUESTED** [review]
Found one issue, but everything else looks good here.

File: src/components/UI/svgs/GopherHomeNodes.tsx
Comment (TylerAPfledderer) [reply src/components/UI/svgs/GopherHomeNodes.tsx:1]: @corwintines ahh indeed! 😅

**Comment by TylerAPfledderer** [comment]
@corwintines change made for the svg in question, and also fixed `GlyphHome` where a fill was being applied when it should be `fill: none`

**Review by corwintines — APPROVED** [review]

**Pull request merged**